starter-code/pizza.js

```diff
@@ -1 +1,42 @@
 // Write your Pizza Builder JavaScript in this file.
+'use strict';
+//$(document).ready(main) tambe per começar
+
+function main () {
+
+  $().removeClass(["sauce-white"]);
+
+  $(".btn-pepperonni").on("click", function(){
+      $(this).toggleClass("active");
+      $(".pep").toggle();
+  });
+
+  $(".btn-mushrooms").click(function(){
+    $(this).toggleClass("active");
+    $(".mushroom").toggle();
+  });
+
+  $(".btn-green-peppers").click(function(){
+    $(this).toggleClass("active");
+    $(".green-pepper").toggle();
+  });
+
+
+/* WE WERE TRYING TO REMOVE THE WHITE
+SAUCE AS DEFAULT
+
+
+  $(".btn-sauce").click(function(){
+    $(this).toggleClass("active");
+    $(".sauce-white").toggle();
```

Comment:
Changing the sauce display is a little bit different than in the previous ones (because you are not toggling the selected class):
```
$(".sauce").toggleClass("sauce-white");
```